### DISCUSSION TITLE
Aurora AU-A1GUZBRGBW coming back unsupported

### DISCUSSION THREAD

**Discussion by zakery292** [Koenkk/zigbee2mqtt#13853] [Devices]
@Koenkk Followed the discussion back in April for the above bulbs and you added support which im most grateful for, aisde from the fact mine appear to be  joining MQTT with a different model number which is causing them to report as un supported devices. how would i go about making them supported. i tried to follow the guide for unsupported items but i got lost and it wont do the logging 

mine seem to come with the model number [RGBGU10Bulb50AU2]

**Comment by Koenkk** [comment]
Added!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by zakery292** [reply]
Thank you!
